src/qml/FeatureForm.qml

```diff
@@ -262,7 +262,12 @@ Page {
           anchors { left: parent.left; right: parent.right }
 
           //disable widget if the form is in ReadOnly mode, or if it's an RelationEditor widget in an embedded form
-          enabled: (form.state !== 'ReadOnly' || EditorWidget === 'RelationEditor' || EditorWidget === 'ValueRelation' || EditorWidget === 'ExternalResource' ) && !!AttributeEditable
+          property bool isEnabled: !!AttributeEditable && (
```

Comment:
I support that you don't control generally `enabled` and make it over `isEnabled` instead.

But I'm wondering if we can even get rid of this variable as well and use just `readOnly` below.

Then we would have `readOnly` when `!AttributeEditable` or the form.state is 'ReadOnly' or it's an embedded `RelationEditor`. Then we control in the widgets, what they are on `readOnly`: Mostly it's `enabled=!readOnly` but e.g. the `RelationEditor` is enabled even on `readOnly` but the control buttons are not visible. What do you think?

Reply:
Not sure if I am the best person to answer this, but I find your arguments reasonable and I love removing complex logic for simpler solutions. However, I would like to do this in a separate PR, so this one can stay focused on clickable links :)

Reply:
Fair enough. Thanks.

Reply:
Wait with it anyway until this is merged #1073 since otherwise you will run into conflicts...